aireview/src/aireview/infrastructure/dust_client.py

```diff
@@ -45,7 +42,6 @@ def create_conversation(self, prompt: str):
             "blocking": True
         }
         response = requests.post(self.url + "/assistant/conversations", headers=self.get_headers(), json=payload)
-        print(response.json())
         response.raise_for_status()
         return self.get_content(response.json())
```

Comment:
**style**: Debug print statements have been properly removed. Consider replacing with proper logging if debugging information is needed.